package.json

```diff
@@ -1,16 +1,18 @@
 {
-  "name": "div-cdd",
+  "name": "div-ccd-e2e-tests",
   "version": "0.0.0",
   "private": true,
+  "description": "",
   "scripts": {
-    // "test-e2e": "NODE_PATH=. codeceptjs run  -c test/end2end --steps "
-  },
-  "dependencies": {
-    "cross-env": "^5.1.3"
+    "pretest": "npm run create:type-definitions",
```

Comment:
Not sure why we need pretest and npm.

Reply:
I added this just to keep definition up to date. The definition file is gitignored anyway.
Again this is to help me with writing tests